person-service/src/main/resources/application.yaml

```diff
@@ -17,7 +17,6 @@ spring:
       patientReporting:
         topic-name: ${PATIENT_REPORTING:nrt_patient}
     dlq:
```

Comment:
For clarity- what would the format for the name of the dlq topic be?

Reply:
It's defined right after that:
```
    dlq:
      retry-suffix: _retry
      dlq-suffix: _dlt
```
So for the patient it's `nrt_person_dlt`

Reply:
Does the sink connector try to sink this into rdb_modern? The regex might try to pick it up, and it usually ends in an error if the table is not there.

Reply:
No, at least it's not supposed to be synchronized with db. DLQ is just for Kafka side of the process.

Reply:
The sink is currently listening for anything topic with the prefix nrt_{} in kafka. Not sure if it will discern this.

Reply:
I see the `"topics.regex": "nrt_[a-zA-Z_]+"`, and that regex will match dlt topics, indeed. If it was `nrt_[a-zA-Z]+` (no underscore included) then only `nrt_patient` would be matched, but not the `nrt_patient_dlt`.
Anyway, that's not a scope for this ticket, because the _dlt suffix was the initial solution for DLQ implemetation. This ticket just extended it for two remaining services, not covered initially.

Reply:
Ah I see. That might be an issue when it gets autodeployed in DTS1. The regex currently accounts for tables like nrt_investigation_confirmation. 
There are a few options: 
- We could hardcode the sink connector's topic list with our post-processing expected set, but it does mean that the list needs to be maintained for all nrt tables that get sinked. 
- Could the dlq's prefix be updated?